src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -140,6 +140,20 @@ public async Task SpecializeAsync()
             _logger.LogDebug("Completed language worker channel specialization");
         }
 
+        public async Task WorkerWarmup()
+        {
+            _logger.LogInformation("Starting Worker warmup");
+            _workerRuntime = _environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName);
+
+            IRpcWorkerChannel rpcWorkerChannel = await GetChannelAsync(_workerRuntime);
+
+            if (_workerRuntime != null && rpcWorkerChannel != null)
+            {
+                await rpcWorkerChannel.SendWorkerWarmupRequest();
+            }
+            _logger.LogDebug("Completed Worker warmup");
```

Comment:
same here
```suggestion
            _logger.LogDebug("Completed worker warmup");
```

Do we want any logs about not sending the request? Right now we log worker warm up complete even when we don't send the worker warm up request